src/test/scala/com/redis/cluster/RedisShardsSpec.scala

```diff
@@ -23,9 +23,21 @@ with BeforeAndAfterAll {
 
   override def beforeEach = {}
 
-  override def afterEach = r.flushdb
+  override def afterEach = try {
+    r.flushdb
+  } catch {
+    case t: Throwable =>
+      t.printStackTrace()
+      //throw t
+  }
 
-  override def afterAll = r.close
+  override def afterAll = try {
+    r.close
+  } catch {
+    case t: Throwable =>
+      t.printStackTrace()
+      //throw t
```

Comment:
Some tests fails on my machine, not in Jenkins, I can't find why, This was a try... I'll revert the change.